src/main/scala/nodes/learning/KernelMatrix.scala

```diff
@@ -25,49 +25,52 @@ trait KernelMatrix {
 
 }
 
-class BlockedKernelMatrix (kernelGen: KernelGenerator, data: RDD[DenseVector[Double]],  train: Boolean, cacheKernel: Boolean) extends KernelMatrix
+class BlockedKernelMatrix (kernelGen: KernelGenerator,
```

Comment:
This is still not the right style. Please see https://github.com/databricks/scala-style-guide#spacing-and-indentation (especially the one about `For classes whose header doesn't fit in a single line`)